src/game/Spell.cpp

```diff
@@ -2595,6 +2595,27 @@ void Spell::SetTargetMap(SpellEffectIndex effIndex, uint32 targetMode, UnitList&
     {
         if (targetMode != TARGET_SELF && targetMode != TARGET_SELF2 && m_spellInfo->Effect[effIndex] != SPELL_EFFECT_SUMMON)
             targetUnitMap.remove(m_caster);
+
+        switch (m_spellInfo->Id)
+        {
+            case 44877:
```

Comment:
Target = TARGET_AREAEFFECT_INSTANT
So you can just insert the creautre id into spell_script_target.